virtio-devices/src/vsock/unix/muxer.rs

```diff
@@ -343,6 +350,11 @@ impl VsockEpollListener for VsockMuxer {
 impl VsockBackend for VsockMuxer {}
 
 impl VsockMuxer {
+    /// connect statement with a space, normally followed by the port and a `\n`
+    const CONNECT_WITH_SPACE_LOWER_CASE: &str = "connect ";
+    /// the minimum connect statement that is still valid
+    const CONNECT_MIN_STATEMENT: &str = "connect 0\n";
```

Comment:
Given these are only used in a single method, there's no need to make them type-level constants. They could be inlined into that method with shorter names (like "prefix" and "min_command"), which would still be sufficiently clear because of the tighter scope.